server/www/templates/posts/media.html

```diff
@@ -1,18 +1,38 @@
+<form>
+    <div class="form-field file">
 
-<form ng-submit="uploadFile()">
-    <div class="form-field">
-      <a ng-href="{{media.original_file_url}}" ng-if="media.original_file_url" target="_blank" translate>post.media.current_url</a>
+        <!-- the image file -->
+        <figure ng-if="media.original_file_url" >
+            <img src="{{media.original_file_url}}" />
+        </figure>
+
+        <div class="form-field">
+
+            <!-- delete button displays only on post edit -->
+            <button class="button-destructive" ng-show="mediaId" ng-click="deleteMedia(mediaId)">
```

Comment:
Styling on these is kinda bad, shouldn't delete and replace be side by side? https://www.dropbox.com/s/ej7rm21fr19mmpo/Screenshot%202016-10-05%2012.50.05.png?dl=0